src/features/unified-share-modal/UnifiedShareModal.js

```diff
@@ -514,44 +517,65 @@ class UnifiedShareModal extends React.Component<Props, State> {
+                    submitting={submitting}
+                    selectedContacts={this.state.inviteCollabsContacts}
+                    suggestedCollaborators={suggestedCollaborators}
+                    updateSelectedContacts={this.updateInviteCollabsContacts}
+                    {...inviteCollabsEmailTracking}
+                >
+                    {this.renderInviteePermissionsDropdown()}
+                    {isInviteSectionExpanded && showUpgradeOptions && this.renderUpgradeLinkDescription()}
+                </EmailForm>
+            </div>
+        );
+
         return (
             <React.Fragment>
-                <Tooltip
-                    className="usm-ftux-tooltip"
-                    isShown={shouldRenderFTUXTooltip && showCalloutForUser}
-                    position="middle-left"
-                    showCloseButton
-                    text={ftuxTooltipText}
-                    theme="callout"
-                >
-                    <div className="invite-collaborator-container">
-                        <EmailForm
-                            contactLimit={contactLimit}
-                            contactsFieldAvatars={avatars}
-                            contactsFieldDisabledTooltip={contactsFieldDisabledTooltip}
-                            contactsFieldLabel={<FormattedMessage {...messages.inviteFieldLabel} />}
-                            getContacts={getCollaboratorContacts}
-                            inlineNotice={inlineNotice}
-                            isContactsFieldEnabled={canInvite}
-                            isExpanded={isInviteSectionExpanded}
-                            isExternalUserSelected={this.hasExternalContact(INVITE_COLLABS_CONTACTS_TYPE)}
-                            onContactInput={this.openInviteCollaborators}
-                            onPillCreate={this.handleInviteCollabPillCreate}
-                            onRequestClose={this.closeInviteCollaborators}
-                            onSubmit={this.handleSendInvites}
-                            openInviteCollaboratorsSection={this.openInviteCollaboratorsSection}
-                            showEnterEmailsCallout={showEnterEmailsCallout}
-                            submitting={submitting}
-                            selectedContacts={this.state.inviteCollabsContacts}
-                            suggestedCollaborators={suggestedCollaborators}
-                            updateSelectedContacts={this.updateInviteCollabsContacts}
-                            {...inviteCollabsEmailTracking}
-                        >
-                            {this.renderInviteePermissionsDropdown()}
-                            {isInviteSectionExpanded && showUpgradeOptions && this.renderUpgradeLinkDescription()}
-                        </EmailForm>
-                    </div>
-                </Tooltip>
+                {recommendedSharingTooltipCalloutName ? (
+                    <Tooltip
```

Comment:
❓ rather than rendering the component twice, does it make sense to just have two objects for their props? example:

```
const recommendedSharingTooltipProps = { ... };
const usmFtuxTooltipProps = { ... };
const tooltipPropsToRender = recommendedSharingTooltipCalloutName ? recommendedSharingTooltipProps : usmFtuxTooltipProps;

<React.Fragment>
  <Tooltip ...tooltipPropsToRender>
    -- actual invited collabs section code rather than pulling it out --
  </Tooltip
</React.Fragment>
```